File: app/src/main/java/acr/browser/lightning/search/engine/StartPageSearch.java
package acr.browser.lightning.search.engine;

import acr.browser.lightning.constant.Constants;

/**
 * The StartPage search engine.
 */
public class StartPageSearch extends BaseSearchEngine {

    public StartPageSearch() {
        super("file:///android_asset/startpage.png", Constants.STARTPAGE_SEARCH);
    }

}

File: app/src/main/java/acr/browser/lightning/search/engine/StartPageSearch.kt
package acr.browser.lightning.search.engine

import acr.browser.lightning.R
import acr.browser.lightning.constant.STARTPAGE_SEARCH

/**
 * The StartPage search engine.
 */
class StartPageSearch : BaseSearchEngine(
        "file:///android_asset/startpage.png",
        STARTPAGE_SEARCH,
        R.string.search_engine_startpage
)
